sections/reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code reviews
+
+At Open Knowledge International (OKI) all code must be reviewed by a colleague before merging into the main branch of the repository (usually `master`). This short document examples how to do code reviews, and why we do it.
+
+## How
+
+The process for code reviews is straight forward as long as you are already following our coding standards for working with version control, and in particular our git flow-influenced process of branch management. The bottom line is that, before any code hits `master`, it must be reviewed by a colleague.
```

Comment:
_must_ is very strong. I'm still have concerns that this stricture doesn't fit in with all projects. Especially in the early stages of a new project.